app/src/features/apiClient/screens/apiClient/components/clientView/components/request/components/KeyValueTable/KeyValueTable.tsx

```diff
@@ -108,7 +116,7 @@ export const KeyValueTable: React.FC<KeyValueTableProps> = ({ data, setKeyValueP
         editable: true,
         onCell: (record: KeyValuePair) => ({
           record,
-          editable: true,
+          editable: record.type !== AUTH_ENTRY_IDENTIFIER,
```

Comment:
We should add `editable` prod in `KeyValueTableProps`. That will keep the auth logic outside without coupling that in this component